### PR TITLE
Use cma plain client

### PR DESCRIPTION
**Pull request by anho**
## Summary

Updating to new major version of CMA SDK and using the plain client. This causes a breaking change in case users consume the `createManagementClient` function.

@ruderngespra is there anything left in regards to documentation inside this repo?

## Todos

-   [x] Implemented feature
-   [ ] Feature with pending implementation

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -82,3 +82,7 @@ workflows:
             - test-unit
             - test-integration
             - test-e2e
+          filters:
```
Comment (anho) [inline .circleci/config.yml:85]: Just making PRs on Circle a bit faster.

File: test/integration/setup.js [hunk 2]
```diff
@@ -8,5 +8,6 @@ const requiredEnvVars = [
 const undefinedEnvVar = (key) => typeof process.env[key] === 'undefined';
 
 if (requiredEnvVars.some(undefinedEnvVar)) {
-  throw new Error('Please define all necessary "CONTENTFUL_INTEGRATION_*" environment variables');
+  console.error('Please define all necessary "CONTENTFUL_INTEGRATION_*" environment variables');
```
Comment (anho) [inline test/integration/setup.js:11]: Just throwing errors was not yielding the actual message to me, I'd attribute it to mocha :)

File: src/bin/cli.ts [hunk 3]
```diff
@@ -15,6 +16,8 @@ import writeErrorsToLog from './lib/write-errors-to-log'
 import { RequestBatch } from '../lib/offline-api/index'
 import { getConfig } from './lib/config'
 import ValidationError from '../lib/interfaces/errors'
+import { PlainClientAPI } from 'contentful-management'
+import { trim } from 'lodash'
```
Comment (marcolink) [inline src/bin/cli.ts:20]: afaik, this is the only way to ensure only relevant imports.
```suggestion
import trim from 'lodash/trim'
```

**Review by marcolink — APPROVED** [review]
LGTM as far as I can judge :)

**Review by suevalov — COMMENTED** [review]
wow, that's the way easier than I expected. is it all that we need to change? basically, almost nothing were used from `contentful-management` and this tool has hardcoded urls for all CMA methods, right?

**Comment by anho** [comment]
@suevalov yes, I was surprised as well.

**Review by suevalov — APPROVED** [review]

**Review by ruderngespra — APPROVED** [review]
lgtm :+1: :)

**Pull request merged**